packages/frontend/src/components/ui/navbar/popovers/preferences/index.tsx

```diff
@@ -27,7 +29,7 @@ export const PreferencesPopover = forwardRef<
     PreferencesPopoverProps
 >(function PreferencesPopover({ open, anchor }, ref) {
     const { t } = useTranslation();
-
+    const chaiId = useChainId();
```

Comment:
Typo here, should be `chainId`

Reply:
Fixed in 79f0dfbb766d3a1ace3dda271e467de26779eebe.